tests/flow/test_results.py

```diff
@@ -161,6 +161,8 @@ def test08_return_duplicate_expression(self):
 
     # Test implicit result-set size limit
     def test09_implicit_resultset_limit(self):
+        # CONFIG command on redis cluster is excluded
+        self.env.skipOnCluster()
```

Comment:
need to check if which default config values can be passed to rlec cluster for redisgraph